sihl-queue/src/repo.ml

```diff
@@ -30,6 +30,14 @@ module type Sig = sig
     -> unit
     -> Sihl.Contract.Queue.instance list Lwt.t
 
+  val search
```

Comment:
@mabiede what do you think?

Reply:
Looks fine, looking forward to test it in the project directly.